workspaces/libnpmpublish/lib/provenance.js

```diff
@@ -18,10 +18,11 @@ const GITLAB_BUILD_TYPE_VERSION = 'v0alpha1'
 const generateProvenance = async (subject, opts) => {
   let payload
   if (ci.GITHUB_ACTIONS) {
-    /* istanbul ignore next - not covering missing env var case */
-    const [workflowPath, workflowRef] = (env.GITHUB_WORKFLOW_REF || '')
-      .replace(env.GITHUB_REPOSITORY + '/', '')
-      .split('@')
+    const relativeRef = (env.GITHUB_WORKFLOW_REF || '').replace(env.GITHUB_REPOSITORY + '/', '')
```

Comment:
Without the `istanbul` comment, the test coverage dropped below 100%. Add this back and we should be good
```suggestion
    /* istanbul ignore next - not covering missing env var case */
    const relativeRef = (env.GITHUB_WORKFLOW_REF || '').replace(env.GITHUB_REPOSITORY + '/', '')
```